templates/partials/header.html

```diff
@@ -89,6 +89,20 @@
               <span translate>Log Out</span>
             </a>
           </li>
+
+          <li role="separator" class="divider" ng-if="!replicationStatus.disabled"></li>
+          <li role="presentation disabled" ng-show="!replicationStatus.disabled && replicationStatus.current !== 'unknown'">
+            <a>
+              Last sync <i class="fa fa-fw fa-refresh"></i><br>
+              <span ng-bind-html="replicationStatus.lastCompleted | relativeDate"></span>
+            </a>
+          </li>
+          <li role="presentation disabled" ng-if="!replicationStatus.disabled" class="sync-status">
+            <a ng-class="replicationStatus.current">
+              <i class="fa fa-fw" ng-class="replicationStatus.icon"></i>
+              <span>{{'sync.status.' + replicationStatus.current | translate}}</span>
```

Comment:
Don't concatenate keys - it makes it really hard to find if they're in use. Instead make `replicationStatus.current` have the full key, or have a `replicationStatus.key`.